src/actions/pause/extra.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+// @flow
+
+import { getSymbols, getSource, getSelectedFrame } from "../../selectors";
+import { isReactComponent, isImmutable } from "../../utils/preview";
+import { findClosestClass } from "../../utils/ast";
+
+import type { ThunkArgs } from "../types";
+
+async function getReactProps(evaluate) {
+  const reactDisplayName = await evaluate(
+    "this.hasOwnProperty('_reactInternalFiber') ? " +
+      "this._reactInternalFiber.type.name : " +
+      "this._reactInternalInstance.getName()"
```

Comment:
This is pre-existing code but I'm confused. The function is called `getReactProps`, but doesn't it just return the component name as a string?

Reply:
woops I skipped over the lines below this. Makes total sense now